test/fuzz/rpc/jsonrpc/server/handler.go

```diff
@@ -26,38 +27,38 @@ func init() {
 	rs.RegisterRPCFuncs(mux, rpcFuncMap, lgr)
 }
 
-func Fuzz(data []byte) int {
+func Fuzz(ctx context.Context, data []byte) error {
```

Comment:
Doesn't the fuzzer rely on the type signature of this function?

Reply:
blast